### PR TITLE
fixes php tags in header

### LINKED ISSUE #7: Header doesn't work on Windows
In the header there are wrong PHP tags so the header is not displayed

### PR DESCRIPTION
**Pull request by k01255322**
fixes #7

### CODE REVIEW

File: header.php [hunk 1]
```diff
@@ -57,11 +57,11 @@
     <div id="wrapper">
 
         <div id="header">
-        	<? include("header_social.php"); ?>
```
Comment (spipau) [inline header.php:60]: Why did you add this PHP? It is working on my side?

Comment (k01255322) [reply header.php:60]: It's not working on Windows

Comment (spipau) [reply header.php:60]: Great, works on my side with Ubuntu

**Review by spipau — APPROVED** [review]
Looks good to me

**Pull request merged**